lib/pkcs7/cryptographer.rb

```diff
@@ -110,9 +135,7 @@ def encrypt(
 
     def verified_signature?(signed_data, public_certificate, ca_store)
       signed_data.verify(
-        [public_certificate],
-        ca_store,
-        nil,
+        [public_certificate], ca_store, nil,
```

Comment:
Done, but there are offenses from RuboCop, especially about class length (> 100 LoC), and I don't know how to better deal with it.

Reply:
You can override the rule in the Rubocop configuration and increased it to 150 or so: `Metrics/ClassLength:` I can check later if I should split the call, because it is getting big 😄

Reply:
Done, with comment to your message as argumentation.

BTW, if you're interested in, I resolved a similar problem via taking off some logic parts to separate modules and include them, example: https://github.com/filewatcher/filewatcher/commit/56e339eb4c00164b727c0d75a66e03873ec68e31#diff-724881cc201748488d7c1b3fc4efec3aedc696877c1d8e228bdc320f197e617c

I can't imagine logical separate parts right now for the `Cryptographer` class, but I think you can if it's possible and in your taste.

But, with chosen by you code style (many arguments, each on separate line) such RuboCop configuration is OK, I think.